README.md

```diff
@@ -1,32 +1,40 @@
-# AOSS - IaC scanning GCB Plugin
+# GCP SCC IaC validation utilities
 
 ## Description
 
-scc-iac-scan-report-utils provide script for handling the result from gcloud scc iac-validation-reports create.
-Currently it offers the following utility files
+This repo provides 2 [go](https://go.dev/) utility scripts for handling the response from [`gcloud scc iac-validation-reports create`](https://cloud.google.com/security-command-center/docs/validate-iac#gcloud-create-iac) command.
+1. SARIF converter
+1. Report validator
 
-## SARIFConverter
+## SARIF converter
 
-SARIFConverter converters the report generated by "gcloud scc iac-validation-reports create" command to a more
-popular SARIF format.
+SARIF Converter converters the response generated by `gcloud scc iac-validation-reports create` command to the industry stardard [SARIF](https://sarifweb.azurewebsites.net/) format. This takes the response from the gcloud command as the input, converts it to the SARIF format and writes the output to a file.
 
-## Validator
+*Example invocation of the script from CLI -*
+```
+go run github.com/google/gcp-scc-iac-validation-utils/SARIFConverter@latest 
+    --inputFilePath=IaCScanReport.json
+    --outputFilePath=IaCScanReport.sarif.json
+```
 
-It checks the scc iac-validation-report against limits set by failure criteria and returns the validation outcome.
+where "IaCScanReport.json" is the report that is generated from the gcloud command and "IaCScanReport.**sarif**.json" is the name of the output file.
 
-- These validation failure criteria could be passed as an expression in a form of input to the script.
+## Report validator
 
-    ``` 'Critical:2,Low:5,Operator:or' ```
+This validates the resopnse generated by `gcloud scc iac-validation-reports create` against thresholds set by "failure_expression" argument to the command. The command returns a success (exit(0)) or fail (exit(1)) code as a result of the validation. The threshold criteria is based on the number of critical, high, medium, and low severity issues that the IaC validation scan encounters.
 
-- If no expression is passed to the scipt following default criteria is used to perform these validation.
+- The failure_expression argument to the command specifies how many issues of each severity are permitted, and also specifies how the issues are aggregated (either AND or OR). For example, if you want the validation to fail if it encounters one critical issue or one high severity issue, set the failure_expression to `'Critical:1,High:1,Operator:OR'`
 
-    ``` 'Critical:1,High:1,Medium:1,Low:1,Operator:or' ```
+- If no expression is passed to the scipt following default criteria is used to perform these validation. `'Critical:1,High:1,Medium:1,Low:1,Operator:OR'` which means that if the IaC validation scan encounters any issue, the build must fail.
```

Comment:
"which means that if the IaC validation scan encounters any issue, the build must fail." 
This feels a little off with the context it is following.

IMO we can write it in a new line like this.
If the expression contains an invalid severity level, operator, or non-whole number severity value, the build will fail.

Reply:
Done, reworded to make the intent clearer.